poc/mastic.py

```diff
@@ -305,15 +308,14 @@ def prep_init(
             onehot_check + counter_check + payload_check,
         ).next(PROOF_SIZE)
 
-        # Concatenate the output shares into one aggregatable output,
-        # applying the FLP truncation algorithm on each FLP measurement
-        # share.
-        truncated_out_share = []
-        for val_share in out_share:
-            truncated_out_share += [val_share[0]] + \
-                self.flp.truncate(val_share[1:])
+        flattened_out_share = []
+        for out_share in out_shares:
+            # The VIDPF transform truncates at every level but the first.
+            if level == 0:
+                out_share = self.truncate(out_share)
+            flattened_out_share += out_share
```

Comment:
`flattened_out_share` --> `truncated_flattened_shares`?

Reply:
Eh, this is a little subjective, but I think "truncated" is implied here.